code/UserForm.php

```diff
@@ -193,7 +193,6 @@ public function getCMSFields()
             // define tabs
             $fields->findOrMakeTab('Root.FormOptions', _t('SilverStripe\\UserForms\\Model\\UserDefinedForm.CONFIGURATION', 'Configuration'));
             $fields->findOrMakeTab('Root.Recipients', _t('SilverStripe\\UserForms\\Model\\UserDefinedForm.RECIPIENTS', 'Recipients'));
-            $fields->findOrMakeTab('Root.Submissions', _t('SilverStripe\\UserForms\\Model\\UserDefinedForm.SUBMISSIONS', 'Submissions'));
```

Comment:
This could've stayed here as all the tabs were created together.

Reply:
I moved it so all submissions related instructions are grouped together.
Seems less cognitive overhead remembering things defined further up the page, to me at least.

It was a step toward this, but as mentioned, decided against it.
>Arguably submission data does not belong here, but this is beyond the scope of this patch.

Reply:
Fair enough. They are not grouped for the other tabs so in the context of this PR it seemed unnecessary. But all good.